src/engine/renderer/Material.cpp

```diff
@@ -1147,6 +1147,38 @@ void MaterialSystem::GenerateWorldCommandBuffer() {
 	GL_CheckErrors();
 }
 
+void MaterialSystem::GenerateDepthImages( const int width, const int height, imageParams_t imageParms ) {
+	int size = std::max( width, height );
+	imageParams_t oldParms = imageParms;
+	imageParms.bits ^= ( IF_NOPICMIP | IF_PACKED_DEPTH24_STENCIL8 );
+	imageParms.bits |= IF_ONECOMP32F;
+
+	depthImageLevels = 0;
+	while ( size > 0 ) {
+		depthImageLevels++;
+		size >>= 1; // mipmaps round down
+	}
+
+	byte* data = new byte[width * height * 4];
```

Comment:
`std::unique_ptr<byte> data( new byte[ width * height * 4 ] )`

Reply:
That can likely be removed with nullptr passed to the function instead, will need to check that later. Either way I don't see the point of creating a wrapper for an allocation that only exists within this function and is freed 15 lines later.

Reply:
Removed `data`, changed this to just pass nullptr instead.